### PR TITLE
Update developers and security section for AWS Device Farm plugin

### PR DESCRIPTION
**Pull request by nikhil-dabhade**
# Description
I am the official maintainer for the AWS Device Farm Jenkins plugin at AWS.
Github Repo: https://github.com/jenkinsci/aws-device-farm-plugin
Plugin page: https://plugins.jenkins.io/aws-device-farm/

1. There is a now dedicated team at AWS Device Farm that maintains the plugin and I would like to add the developers to the plugin. 
2. I am also requesting to remove members that are no longer part of the maintainer team. 
3. I would also like to remove members that were added when the plugin state was marked abandoned some time back and additional maintainers were added for an urgent release. 

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by patcadelina** [comment]
All good, thanks @nikhil-dabhade for taking ownership.

**Comment by slide** [comment]
Need @daniel-beck to weigh in because there is a change to security contact informaton.

**Comment by slide** [comment]
Please also resolve the conflicts.

**Comment by daniel-beck** [comment]
This security section is redundant as the same user is a maintainer, and the user is backed by the same email address.

File: permissions/plugin-aws-device-farm.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ github: "jenkinsci/aws-device-farm-plugin"
...
 developers:
-- "chabom"
-- "oleg_nenashev"
-- "patcadelina"
+- "nikdabn"
+- "aws_device_farm_jenkins"
+security:
+  contacts:
+    jira: aws_device_farm_jenkins
+    email: "aws-device-farm-jenkins@amazon.com"
```
Comment (slide) [inline permissions/plugin-aws-device-farm.yml:12]:
```suggestion
```

**Comment by nikhil-dabhade** [comment]
@oleg-nenashev @patcadelina Thanks for the update. 
@slide Thanks for including @daniel-beck and the feedback. I am resolving it now. 
Also will update the security section.

**Comment by nikhil-dabhade** [comment]
The error in check says 
"User name not known to Artifactory: aws_device_farm_jenkins"

This is a valid account on jenkins site. 
@oleg-nenashev Any suggestions ?

**Comment by nikhil-dabhade** [comment]
@slide @daniel-beck I update the security section with AWS security contact.

File: permissions/plugin-aws-device-farm.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ paths:
 - "org/jenkins-ci/plugins/aws-device-farm"
 developers:
 - "nikdabn"
-- "chabom"
-- "oleg_nenashev"
-- "patcadelina"
+- "aws_device_farm_jenkins"
+security:
+  contacts:
+    email: "aws-security@amazon.com"
```
Comment (slide) [inline permissions/plugin-aws-device-farm.yml:11]:
```suggestion
```

Is the email address associated with aws_device_farm_jenkins aws-security@amazon.com? If so, the security section is not needed at all.

**Comment by slide** [comment]
Has the aws_device_farm_jenkins user logged in at least once to Artifactory? https://repo.jenkins-ci.org/webapp/#/home

File: permissions/plugin-aws-device-farm.yml (hunk 2 above)
Comment (daniel-beck) [reply permissions/plugin-aws-device-farm.yml:11]: This is a known contact for other plugins maintained by Amazon AWS folks. I approve.

(Also not the same email address.)

Comment (nikhil-dabhade) [reply permissions/plugin-aws-device-farm.yml:11]: @slide No. The email associated with aws_device_farm_jenkins is different. 
Thank you @daniel-beck

**Comment by nikhil-dabhade** [comment]
@slide We just checkout out the login at our end. The user logged in today.

**Comment by nikhil-dabhade** [comment]
Thank you @slide !

**Pull request merged**